db/models.py

```diff
@@ -14,3 +14,44 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movies_actor")
+    genres = models.ManyToManyField(Genre, related_name="movies_genre")
```

Comment:
The `related_name='movies_genre'` is inconsistent with typical naming conventions. Consider using `related_name='movies'` to reflect the relationship from `Genre` to `Movie` more clearly.